IAteIt/IAteIt/View/Feed/Component/FeedHeaderView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  PhotoCardView.swift
+//  IAteIt
+//
+//  Created by Beone on 2023/03/07.
+//
+
+import SwiftUI
+
+struct FeedHeaderView: View {
+    
+    let profilePicSize: CGFloat = 36
+    var PostInfo: TempFeedHeader
+    var tempPlateList = TempPlateData().list
+    
+    var body: some View {
+        VStack {
+            HStack(alignment: .top) {
```

Comment:
```suggestion
            HStack(alignment: .center, spacing: 12)
```
이렇게 하고 아래 아이디/장소/시간 VStack의 padding leading 12, top 2를 빼는 게 중앙정렬/간격 맞출 수 있어 좋을 것 같습니다. 
SwiftUI padding이 피그마와 동일하게 줘도 프리뷰에는 다르게 나와서 맞추기가 어렵네요..

Reply:
감사합니다 수정했습니다!!